pkg/kn/commands/version/version.go

```diff
@@ -26,19 +28,22 @@ var BuildDate string
 var GitRevision string
 var ServingVersion string
 
-// VersionsAPIs hold the list of supported versions and APIs for component kn can work with
-type VersionsAPIs struct {
-	Versions, APIs []string
-}
+type SupportedAPIs []string
 
 // update this var as we increase the serving version in go.mod
 var knServingDep = "v0.8.0"
-var supportMatrix = map[string]*VersionsAPIs{
-	knServingDep: {[]string{"v0.8.0", "v0.7.1"}, []string{"v1alpha1"}},
+var supportMatrix = map[string]*SupportedAPIs{
+	knServingDep: {"serving.knative.dev/v1alpha1 (knative-serving v0.8.0)"},
```

Comment:
Could this be read from external file?

Reply:
I think yes, but then we'd need to update the external file when we bump the version, so I think updating it directly here would save us introducing an additional file. Also, as per discussion above, I think it makes more sense to hard code it now as feeding the var in here doesn't add much value and one has to anyway edit this file part of release.